tests/unit/integrations/admin/background-indexing-integration-test.php

```diff
@@ -174,64 +217,198 @@ public function test_constructor() {
 	 */
 	public function test_register_hooks() {
 		Monkey\Actions\expectAdded( 'admin_init' );
+		Monkey\Actions\expectAdded( 'wpseo_indexable_index_batch' );
```

Comment:
Normally I think we always use has( instead of expectAdded, not sure if there is a reason you chose to use this except that there was already a use here.

Reply:
It's something that I moved from [the original PR](https://github.com/Yoast/wordpress-seo/pull/17989), but I think it is because we already have been using `expectAdded` here, so for uniformity reasons?